pycqed/instrument_drivers/meta_instrument/qubit_objects/CCL_Transmon.py

```diff
@@ -1803,6 +1803,7 @@ def measure_qubit_frequency_dac_scan(self, freqs, dac_values,
                                          pulsed=True, MC=None,
                                          analyze=True, fluxChan=None, close_fig=True,
                                          nested_resonator_calibration=False,
+                                         nested_resonator_calibration_use_min=False,
```

Comment:
Change does not belong in this PR but I'm fine with merging.

Reply:
Agreed